### DISCUSSION TITLE
New in pre-release: `CmdZoomCustom` takes an argument

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#4411] [Announcements]
You can now create a [keyboard shortcut](https://www.sumatrapdfreader.org/docs/Customizing-keyboard-shortcuts) for arbitrary zoom levels by providing an argument to `CmdZoomCustom` [command](https://www.sumatrapdfreader.org/docs/Commands#cmdzoomcustom).

Available in latest [pre-release](https://www.sumatrapdfreader.org/prerelease) build.

**Comment by ramiropistoia** [comment]
Hello, when will version 3.6 be released?

**Reply by kjk** [reply]
don't know. pre-release is just as good as 3.5.2 (most of the time)

**Reply by GitHubRulesOK** [reply]
most of the time pre-release is better than 3.5.2

**Reply by Conflituoso** [reply]
Hi, 32bit only in the final version?

**Reply by GitHubRulesOK** [reply]
@Conflituoso unsure what your asking 32bit is available daily but has a few differences to 64 bit and not shown 
standalone is currently https://www.sumatrapdfreader.org/dl/prerel/16515/SumatraPDF-prerel-32.exe
similar for installer version but beware 32bit needs close exe to make many "advanced setting" edits.

**Reply by Conflituoso** [reply]
Thank you very much for the explanation.
Only 64-bit versions are available on this page - https://www.sumatrapdfreader.org/prerelease
Sorry for my English...
All the best.
